btap/src/feature_selection.py

```diff
@@ -83,20 +87,21 @@ def select_features(args, estimator_type='lasso', min_features=20):
     data_json = json.dumps(data).encode('utf-8')
 
     # copy data to minio
-    acm.access_minio(operation='copy',
+    write_to_minio = acm.access_minio(operation='copy',
                      path=args.output_path,
                      data=data_json)
-
+    logger.info("write to mino  ", write_to_minio)
```

Comment:
Missing placeholder for data.

```suggestion
    logger.info("write to mino %s", write_to_minio)
```